packages/back-end/test/data-generator/new-generator.ts

```diff
@@ -2,11 +2,7 @@
 import fs from "fs";
 import { GrowthBook } from "@growthbook/growthbook";
 import { jStat } from "jstat";
-
-// TODO: set seeds to enable replicable data
-
-const NUM_USERS = 10000;
-const OUTPUT_DIR = "/tmp/csv";
+import { int } from "aws-sdk/clients/datapipeline";
```

Comment:
This is likely a typo.  I'm guessing you meant to use `number` instead of `int` as a type somewhere and your IDE autocompleted to this

Reply:
yeah i can't shake the urge to `int`. fixed.